.github/workflows/docker-publish.yml

```diff
@@ -59,6 +59,12 @@ jobs:
         vars=$(cat build.json | jq -c '[.variable | to_entries[] | {"key": .key, "value": .value.default}] | from_entries')
         echo "vars=$vars" | tee -a "${GITHUB_OUTPUT}"
 
+    - id: get-version
+      if: ${{ github.ref_type == 'tag'  && startsWith(github.ref_name, 'v') }}
+      run: |
+        tag="${{ github.ref_name }}"
+        echo "AIIDA_VERSION=${tag#v}" >> $GITHUB_OUTPUT
```

Comment:
I assume `${tag#v}` is drop `v` in front, correct?

Reply:
Yep.